apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -1149,14 +1215,10 @@ private RifFile editSampleRecords(
       List<List<List<String>>> editedRifRecords =
           records.getRecords().map(editor).collectList().block();
 
-      // Build a CSVFormat with the specific header needed for the RIF file type.
-      String[] csvHeader =
-          Stream.concat(
-                  Stream.of("DML_IND"),
-                  Arrays.stream(rifFileEvent.getFile().getFileType().getColumns())
-                      .map(e -> e.name()))
-              .toArray(String[]::new);
-      CSVFormat csvFormat = RifParsingUtils.CSV_FORMAT.withHeader(csvHeader);
+      // Get the header from the original file
+      List<String> csvHeader = RifParsingUtils.createCsvParser(inputFile).getHeaderNames();
```

Comment:
Instead of trying to create a new header with everything in the schema, we instead pull the header from the original file since the order could be different.